specification/components/schemas/eRS-FetchWorklist-List.yaml

```diff
@@ -976,4 +908,248 @@ properties:
+                        | ------------------------------ | ----------- | -------------------------------------------------------------------------------------------------------------------------- |
+                        | `patient`                      | `1..1`      | The patient associated with the `ReferralRequest`                                                                          |
+                        | `priority`                     | `1..1`      | The priority associated with the `ReferralRequest`                                                                         |
+                        | `referralCreated`              | `1..1`      | The Date and Time the `ReferralRequest` was initiated                                                                      |
+                        | `clinicalContext`              | `1..1`      | The clinical context for the `ReferralRequest`                                                                             |
+                        | `referrer`                     | `1..1`      | The `Practitioner` who initially creates the `ReferralRequest`                                                             |
+                        | `referralLastUpdated`          | `1..1`      | The `dateTime` the `ReferralRequest` was last updated                                                                      |
+                        | `status`                       | `1..1`      | The status of the `ReferralRequest`                                                                                        |
+                        | `reason`                       | `0..1`      | The reason the `ReferralRequest` is on the Awaiting Booking worklist, only included if `status` is `APPOINTMENT_CANCELLED_BY_PROVIDER` or `APPOINTMENT_CANCELLED_BY_PATIENT` |
+                      type: array
+                      items:
+                        anyOf:
+                          - $ref: './extensions/worklists/Extension-WorklistItem-Patient.yaml'
+                          - $ref: './extensions/worklists/Extension-WorklistItem-Priority.yaml' 
+                          - $ref: './extensions/worklists/Extension-WorklistItem-ReferralCreated.yaml'
+                          - $ref: './extensions/worklists/Extension-WorklistItem-ClinicalContext.yaml'
+                          - $ref: './extensions/worklists/Extension-WorklistItem-Referrer.yaml'
+                          - $ref: './extensions/worklists/Extension-WorklistItem-ReferralLastUpdated.yaml' 
+                          - $ref: './extensions/worklists/Extension-WorklistItem-Status.yaml'
+                          - title: reason
+                            type: object
+                            required:
+                              - url
+                              - valueCodeableConcept
+                            properties:
+                              url:
+                                type: string
```

Comment:
as above enum and example should be "reason"

Reply:
Updated